solr/test-framework/src/java/org/apache/solr/util/TestHarness.java

```diff
@@ -453,6 +455,22 @@ public LocalRequestFactory() {}
      */
     @SuppressWarnings({"unchecked"})
     public LocalSolrQueryRequest makeRequest(String... q) {
+      return makeRequest(false, q);
+    }
+
+    /**
+     * Creates a LocalSolrQueryRequest based on variable args. Note that tests that want to test
+     * multi-threading should ensure that multiple commits are used and that the merge policy will
+     * leave a suitable number of segments.
+     *
+     * @param testMultiThreadedRandom if true 50% of requests will include the multiThreaded=true
+     *     parameter.
+     * @param q the parameters for the request.
+     * @see #makeRequest(String...)
+     * @return a LocalSolrQueryRequest based on the supplied parameters
+     */
+    @SuppressWarnings("unchecked")
+    public LocalSolrQueryRequest makeRequest(boolean testMultiThreadedRandom, String... q) {
```

Comment:
TestHarness might not yet be deprecated but I'd prefer we treat it as such.  I should deprecate it.
https://issues.apache.org/jira/browse/SOLR-11872
Even if SOLR-11872 fails to go anywhere, I also object to this new method here.